### PR TITLE
Dead code: release() method

### PR DESCRIPTION
**Pull request by giorgiosironi**
This is not used at the moment, and in general we have started to log errors and remove the message from the queue rather than letting it there for a possibly infinite number of retries

### CODE REVIEW

**Comment by stephenwf** [comment]
Yeh that makes more sense, is there a case where we need to release the items again? I suppose since we have persistence for Gearman it shouldn't matter

**Comment by giorgiosironi** [comment]
We haven't encountered temporary errors yet, but if it happens we can avoid removing the element from the queue and it will be added back automatically at the end of the visibility timeout.

**Comment by giorgiosironi** [comment]
There is also `Console::ramlCommand()`, given it is managed with Composer now should we remove that too?

**Comment by stephenwf** [comment]
I think theres a few things in console that were there to speed up further development of commands from the eLife Silex Starter, such as `hello` `echo` `spawn` `debug:search:random` and `debug:search`

**Comment by giorgiosironi** [comment]
Are they necessary now?

**Comment by stephenwf** [comment]
No I don't think so

File: tests/src/Search/ExampleWebTest.php [hunk 1]
```diff
@@ -12,11 +12,8 @@ class ExampleWebTest extends ElasticTestCase
      */
     public function testCanRunCommand()
     {
-        // Run command
-        $lines = $this->runCommand('hello');
-        // And test.
-        $this->assertEquals('Hello from the outside (of the global scope)', $lines[0]);
-        $this->assertEquals('This is working', $lines[1]);
+        $lines = $this->runCommand('help');
+        $this->assertGreaterThan(1, count($lines));
```
Comment (stephenwf) [inline tests/src/Search/ExampleWebTest.php]: 👍  Forgot about that test that uses it, good catch

**Pull request merged**